packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -69,10 +69,6 @@ export class SpinButton extends BaseComponent<ISpinButtonProps, ISpinButtonState
   private _lastValidValue: string;
   private _spinningByMouse: boolean;
 
-  private _onValidate?: (value: string) => string | void;
-  private _onIncrement?: (value: string) => string | void;
-  private _onDecrement?: (value: string) => string | void;
-
   private _currentStepFunctionHandle: number;
   private _initialStepDelay = 400;
```

Comment:
This all seems reasonabvle! Definitely a bug to assume props don't change in the constructor.

Can you add a change file? (run `npm run change` from the root.)

Note that you may want to have an "upstream" remote that points to (case sensitive)

`https://github.com/OfficeDev/office-ui-fabric-react.git`

Reply:
Sorry. I'm not sure how to do this. It says `No change file is needed`.

Reply:
@RoelVB https://help.github.com/articles/configuring-a-remote-for-a-fork/

I will do it for you here.